packages/common-all/src/types/workspace.ts

```diff
@@ -26,6 +26,10 @@ export enum DVaultSync {
   SYNC = "sync",
 }
 
+export type DVaultUriVariant = DVault & {
```

Comment:
will this be the default moving forward?

Reply:
I think this can probably be removed actually.  Everything should be able to be formulated off a URI variant of wsRoot + the relative path on a `DVault`.  

Unless sometimes `DVaults` have absolute paths? In which case it'll be problematic for the web extension.